.travis.yml

```diff
@@ -2,7 +2,7 @@ language: ruby
 rvm:
   - 1.9.3
   - 2.0
-  - 2.1
-  - 2.2
-  - 2.3.0
```

Comment:
I think we need to just drop the bugfix/patch digit for `2.3.0` to use the latest version.  See the build [here](https://travis-ci.org/NARKOZ/gitlab/jobs/160698118) it uses `2.2.5` not `2.2.0`.   The Travis docs also indicate that `2.1`, `2.2`, etc., are aliases: https://docs.travis-ci.com/user/languages/ruby#Choosing-Ruby-versions-and-implementations-to-test-against.